_opensearch/query-dsl/geo-and-shape.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Geographic and shape queries
+parent: Query DSL
+nav_order: 70
+---
+
+# Geographic and shape queries
+
+You can search multiple dimensions with the geographic and shape query types. You can create queries to search two-dimensional geometries that map out cartesian data such as `point` fields that support x and y pairs and `shape` fields that support points, lines, circles and polygon shapes.
+
+## Geographic queries
+
+You can use the following geographic queries to search documents that include `geo_point` field data:
+
+- **Geo-bounding box** `geo_bounding_box` – Returns documents with `geo_point` field values that are within a bounding box.
+- **Geo distance** `geo_distance` – Returns documents that contain the geographic points within a specified distance from a central point.
+- **Geo polygon** `geo_polygon` – Returns documents that contain the specified geographic points within the specified polygon.
+- **xy point queries** – Returns documents that contain two-dimensional coordinates in `xy_point` or `xy_shape` fields.  To learn more, see [xy queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/xy/).
+
+## Pre-indexed shape query
+
+You can use the pre-indexed shape query `indexed_shape` to specify a `shape` query for a shape that has been previously been indexed. You need to specify the following pre-indexed shape fields `id`, `index` `path` and `routing`. Use the `ignore_unmapped` option set to `true` to ignore any fields not mapped to the previous index specified in the `indexed_shape` query. By default, `ignore_unmapped` is set to `false` to throw an exception if the field is not mapped.
+
+### Spacial relation operators
```

Comment:
```suggestion
### Spatial relation operators
```

Reply:
I would remove this section because it is described in the xy queries document, and probably will be described in the geoshape queries document. If we keep this section, we should make sure that both query types use the same parameters.

Reply:
Additionally, spatial relation operators can be used without a pre-indexed shape at least in xy queries, so if we keep this section, it should become the same level header as pre-indexed shape.